packages/dds/tree/src/test/feature-libraries/modularChangeFamilyIntegration.spec.ts

```diff
@@ -193,19 +190,15 @@ describe("ModularChangeFamily integration", () => {
 				]),
 			};
 
-			const fieldBExpected = purgeUnusedCellOrderingInfo([
+			const fieldBExpected: SF.Changeset = [
 				{ count: 1, changes: nodeId2 },
-				// The two marks below a not essential and only exist because we're currently using tombstone
+				// The two marks below a not essential and only exist because we're using tombstones
```

Comment:
Not related to your changes, but we might as well fix this.

```suggestion
				// The two marks below are not essential and only exist because we're using tombstones
```